plugin.video.iplayerwww/addon.xml

```diff
@@ -1,7 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.iplayerwww" name="iPlayer WWW" version="1.2.10" provider-name="CaptainT, Cas, ihurst, primaeval">
+<addon id="plugin.video.iplayerwww" name="iPlayer WWW" version="1.3.0" provider-name="CaptainT, Cas, ihurst, primaeval">
   <requires>
-    <import addon="xbmc.python" version="2.1.0"/>
+    <import addon="xbmc.python" version="2.19.0"/>
+    <import addon="xbmc.gui" version="5.3.0"/>
+    <import addon="xbmc.json" version="6.20.0"/>
+    <import addon="xbmc.addon" version="14.0.0"/>
```

Comment:
you don' t need those as you only use python. Please drop them and make sure your PR only contains a single commit